Packs/Cryptosim/IncidentFields/incidentfield-Cryptosim_Log_Type.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@haldunerbay please add a prefix to the cliname, id and name of the field. Also, please associate it with the cryptosim incident type and not with all of the types.

Reply:
Added.